serving/docs/lmi/configurations_large_model_inference_containers.md

```diff
@@ -75,7 +75,7 @@ If you specify Engine to be MPI, rolling_batch to auto or lmi-dist in DeepSpeed
 |--------------------------------------------------------------------|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------|
 | option.quantize	                                                   | No	       | Use `option.quantize` quantize technology to the model. gptq quantize requires to load a gptq model. `bitsandbytes` is deprecated, use `bitsandbytes8` instead, both the options are the same.  	                            | `bitsandbytes8, gptq` <br/> Default: `None`	 |
 | Advanced parameters	                                               |
-| option.paged_attention	                                            | No	       | Use PagedAttention or not. Default is always use. Disable this if you plan to run on G4 or older GPU architecture	                                                                                                           | Default: `true`	                             |
+| option.paged_attention [Deprecated since 0.26.0]	                  | No	       | Use PagedAttention or not. Default is always use. Disable this if you plan to run on G4 or older GPU architecture	                                                                                                           | Default: `true`	                             |
```

Comment:
we should just remove this option, it is not used anymore

Reply:
Removed